GVRf/Framework/framework/src/main/java/org/gearvrf/GVRShaderTemplate.java

```diff
@@ -14,6 +14,9 @@
  */
```

Comment:
No real changes to this file. Please revert to previous version. You can do that with ```git checkout HEAD~1 GVRShaderTemplate.java```